### DISCUSSION TITLE
Homarr update failed

### DISCUSSION THREAD

**Discussion by Narkowii** [tteck/Proxmox#4007] [Help Needed]
hello; update failed v0.15.5 to v0.15.6

`-Updating 59e5c030..5b23f7d1   
error: Your local changes to the following files would be overwritten by merge:
        package.json
Please commit your changes or stash them before you merge.
Aborting
Already up to date.
 \No update required.`

**Comment by havardthom** [comment]
Did you do any local changes to `/opt/homarr/package.json`? What is the output if you run `git diff` in `/opt/homarr` directory? probably need to revert those local changes before updating
